### ISSUE TITLE
The whole AR scene jumps to the side 10-15 sek after placement

### ISSUE THREAD

**Issue by llllkatjallll** [needle-tools/needle-engine-support#176]
### Describe the bug 💬

After placing a simple AR scene (via clicking on the white circle on the floor/table) the scene stays on the right position, but after 10-15 seconds it disappears and appears ca. 1-1.5 meter away from the previos position, on the right side. 
I testet it on several Android phones with good light condition, it still happened every time.

### Operating System 👩‍💻

Windows

### What browsers are you seeing the problem on? 🏄‍♂️

Chrome

### Editor Version 🎲

2021.3

### Needle Exporter version

3.11.6

### Project Info (Unity only) 📜

```json
{
  "ExportInfoGameObjectName": "Export Info",
  "ExportInfoGameObjectIsEnabled": true,
  "UnityProjectPath": "E:/GIT/immersive-hackathon-2023/development/Immersive Hackathon 2023/Assets",
  "UnityVersion": "2021.3.29f1",
  "SceneName": "BugreportScene",
  "ProjectPath": "Needle/SkyCreatorNeedleScene",
  "ProjectDirectoryExists": true,
  "ProjectIsInstalled": true,
  "NeedleEngineInstalled": true,
  "HasNodeInstalled": true,
  "NodeVersion": "v16.15.0",
  "NpmVersion": "8.5.5",
  "TypescriptVersion": "'tsc' is not recognized as an internal or external command,; ; operable program or batch file.; ; ",
  "HasTokTxInstalled": true,
  "HasMinimumToktxVersionInstalled": false,
  "RenderPipeline": "URP-HighFidelity (UnityEngine.Rendering.Universal.UniversalRenderPipelineAsset)",
  "GzipEnabled": false,
  "NeedleEngineExporterVersion": "3.11.6",
  "NeedleEngineVersion": "3.11.6",
  "NeedleEngineSamplesVersion": null,
  "NeedleEngineExporterPath": "Packages/com.needle.engine-exporter/package.json",
  "NeedleEnginePath": "E:/GIT/immersive-hackathon-2023/development/Immersive Hackathon 2023/Assets/../Needle/SkyCreatorNeedleScene/node_modules/@needle-tools/engine/package.json",
  "FileStats": "By Type:\r\n•\t.glb →  x 1 = 0,6 mb\r\n•\t<b>Total = 0,6 mb</b>\r\n----\r\nBy Directory:\r\n•\tassets →  x 1 = 0,6 mb\r\n",
  "NeedleComponentsInScene": [
    "Needle.Engine.ExportInfo",
    "Needle.Engine.Codegen.ComponentGenerator",
    "Needle.Engine.Deployment.DeployToGlitch",
    "Needle.Engine.Components.GltfObject",
    "Needle.Engine.Components.ObjectRaycaster",
    "Needle.Engine.Components.WebARSessionRoot",
    "Needle.Engine.Components.OrbitControls",
    "Needle.Engine.Components.WebXR"
  ],
  "TypeCacheIsDirty": false,
  "TypeScriptTypes": []
}
```

### Steps to reproduce 🔢

```bash
1) Create an object that should appear in AR and put it in AR Session root on position (0,0,0).
2) Export the project and open it with Android phone/tablet. ( or open my bugreport website https://bugreport-ar-placement.glitch.me/)
3) Start AR mode, place the scene in front of you. 
4) Wait for 5-20 sec, look at the AR object.
5) Expected: suddenly object disappears and appears on your right
```

### Reproduction ♻

https://bugreport-ar-placement.glitch.me/

I also uploaded the unity-project with the reproduction.

### Validations 🩹

- [X] Follow our [Code of Conduct](https://github.com/needle-tools/needle-engine-support/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://docs.needle.tools).
- [X] Check that there isn't [already an issue](https://github.com/needle-tools/needle-engine-support/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Needle-Engine-specific issue. For example if it's a Unity related bug, it should be reported to [Unity](https://unity3d.com/unity/qa/bug-reporting) instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/needle-tools/needle-engine-support/discussions) or join our [Discord Chat Server](https://discord.needle.tools).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by marwie** [comment]
thanks for the report. reproduced on android with the vite template

**Comment by marwie** [comment]
I see the same issue in three.js' AR example https://threejs.org/examples/webxr_ar_hittest.html
and also had a few tests where modelviewer.dev did disappear and then re-appear
I also tested with Needle Engine versions going 3 months back and it consistently happened (using three 146) so it might perhaps be a chrome bug

**Comment by marwie** [comment]
Opened an issue in three to get more information https://github.com/mrdoob/three.js/issues/26686

**Comment by marwie** [comment]
It's a chromium bug: https://bugs.chromium.org/p/chromium/issues/detail?id=1475680

**Comment by marwie** [comment]
Fixed in 3.19.1
